theseus_gui/src/components/ui/Navigation.vue

```diff
@@ -0,0 +1,155 @@
+<script setup>
+import { SearchIcon, BookIcon, ClientIcon, PlusIcon, SettingsIcon } from 'omorphia'
+import { useInstances } from '@/store/state'
+import { RouterLink, useRoute } from 'vue-router'
+import SideNav from '@/components/SideNav.vue'
+import UserSection from '@/components/ui/UserSection.vue'
+import Instance from '@/components/ui/Instance.vue'
+const instances = useInstances()
+
+instances.fetchInstances()
+const route = useRoute()
+</script>
+
+<template>
+  <SideNav class="navigation">
+    <UserSection />
+    <div class="pages-list">
+      <RouterLink
+        to="/"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <ClientIcon />Home</RouterLink
+      >
+      <RouterLink
+        to="/browse"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <SearchIcon />Browse</RouterLink
+      >
+      <RouterLink
+        to="/library"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <BookIcon />Library</RouterLink
+      >
```

Comment:
For these- I would diverge from the standard button style to be closer to the design. Similar to spotify's layout, the icons should be bigger with the text more spaced out (also refer to the design for this and below)
![image](https://user-images.githubusercontent.com/18202329/224780026-1bba00fb-39af-4d3b-bbaf-f3e5c478bb08.png)

The "active" state also isn't working, where the current page is slightly greyed out

Reply:
Let me know if the button layout looks better. Added active state to the router.
There isn't any active class in Omorphia it seems, so I added some stuff to the component for now.